app/views/article/_index_default.html.erb

```diff
@@ -22,7 +22,14 @@
 <% if document.access_panels.online? && !document.eds_restricted? %>
   <ul class='document-metadata dl-horizontal dl-invert'>
     <% document.access_panels.online.links.each do |link| %>
-      <li><%= link_to(link.text, link.href) %></li>
+      <li>
+        <% if link.text =~ /^Find it in print/ -%>
```

Comment:
Yeah, I had thought about that too, but I didn't want to modify the SearchWorks Link class. It's an "ILL" category link, so we could add that as a new thing to the `SearchWorks::Links::Link` class...